index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/fddcccfd38.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Local TubeClone</title>
+</head>
+
+<body>
+    <!--header start-->
+    <header>
+        <nav class="flex-container">
+            <i class="fas fa-bars"></i>
+            <img class="logo" src="images/logo" alt="image of logo">
+            <form action="">
+                <input class="myInput" type="text" id="search" placeholder="Search"><i class="fas fa-search"></i>
+            </form>
+
+            <i class="fas fa-upload"></i>
+            <button class="sign" type="submit">Sign in</button>
```

Comment:
- [ ] Your navbar is not centered. Please fix this. **Hint** since you are using flex's space-between, this is an easy fix, you can simply put this 2 items together and they will be moved to the right, allowing your form to stay in the center
![image](https://user-images.githubusercontent.com/53256139/91039621-7eb67100-e5d2-11ea-9446-4e950fc80c89.png)

![image](https://user-images.githubusercontent.com/53256139/91039604-75c59f80-e5d2-11ea-9e36-67dc6eca00a5.png)